### ISSUE TITLE
Max Downloads

### ISSUE THREAD

**Issue by downdrown** [downtuned42/ds-dec#4]
Maybe check for a "Number of Max downloads exceeded"-Message and display an error instead of error response!

**Comment by downtuned42** [comment]
Indeed, known errocodes/messages  should be translated to more readable messages.
